deploy/kubernetes/rbdplugin.yaml

```diff
@@ -0,0 +1,120 @@
+# This YAML defines all API objects to create RBAC roles for csi node plugin.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-nodeplugin
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: csi-nodeplugin
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list"]
```

Comment:
does `csi-nodeplugin` need these privileges at the cluster level?

Reply:
@bassam Yes, since secrets can be located in different namespaces, we need ability to access secrets in a cluster wide manner.

Reply:
@sbezverk seems like a security issue to me. I'd love us to explore a different design for managing secrets.

Reply:
@bassam I would be very interested in participating in the proposed effort.

Reply:
+1 @bassam 

ClusterRole is too wide, should be a Role within a namespace. rbd secrets are created in that namespace.